README.md

```diff
@@ -55,22 +51,19 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+# 📖 [Microverse Hello world] <a name="about-project"></a>
 
-**Hello world** is a...
+**[Mcroverse Hello world]** is a simple project. It is the first project created as part of Microverse curriculum.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+>HTML & CSS
+>Linters
 
 <details>
   <summary>Client</summary>
```

Comment:
- [ ] The tech stack that you used is HTML and CSS. Please remove the REACTJS, EXpress, and Postgress below. 

```html

<details>
  <summary>Client</summary>
  <ul>
    <li><a href="#">React.js</a></li> <!-- Replace this REACT with HTML -->
   <li><a href="#">CSS</a></li> <!-- Add a new line with CSS -->
  </ul>
</details>
<!-- Remove from here -->
<details>
  <summary>Server</summary>
  <ul>
    <li><a href="https://expressjs.com/">Express.js</a></li>
  </ul>
</details>
<details>
<summary>Database</summary>
  <ul>
    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
  </ul>
</details>
```